### PR TITLE
Dev branch d

### PR DESCRIPTION
**Pull request by mitalraithatha**
Upon professor request again creating new pull request for DEV_BRANCH_D.
.
Reviewers ,can you please look into this pull request and again review for the same?
Thanks

### CODE REVIEW

**Review by navid-hossain — COMMENTED** [review]
Indentation problem in format_convert.cpp like before. spacing and indentations are not correct. Other than that, everything looks good. The code is running and the output files are updating after running the simulation.

**Review by tarimislam — COMMENTED** [review]
Indentation problem same like task b otherwise everything looks good

File: src/top_model/format_convert.cpp [hunk 1]
```diff
@@ -0,0 +1,211 @@
...
+
+/**
+* get components from the original generated output file (unformatted file)
+* in the same line
+*/
+
+void getcomponent(char* data, char* component) {
+    int i,j;
+    char *temp;
+	temp = strstr(data,model);
```
Comment (navid-hossain) [inline src/top_model/format_convert.cpp:35]: check indentation please

**Comment by mitalraithatha** [comment]
Hello,
Can you please check the same in your local.
In local it will definitely come.

File: src/top_model/format_convert.cpp [hunk 2]
```diff
@@ -0,0 +1,211 @@
...
+*/
+bool getvalue(char *data, char *value) {
+	char * temp;
+	int i,j;
+	temp = strstr(data,"{");
+	if(temp[1] == '}') {
+		return false;
+	}
+	for(i=1,j=0;temp[i]!='}';i++,j++) {
+		value[j] = temp[i];
```
Comment (navid-hossain) [inline src/top_model/format_convert.cpp:70]: check indentation please... should be 4 spaces... use 4 spaces instead of using TAB

**Review by navid-hossain — COMMENTED** [review]
> Hello,
> Can you please check the same in your local.
> In local it will definitely come.

We checked. You are using tab that's why its showing like this here. If you use 4 spaces instead of tab it will be fixed.

**Comment by mitalraithatha** [comment]
> > Hello,
> > Can you please check the same in your local.
> > In local it will definitely come.
> 
> We checked. You are using tab that's why its showing like this here. If you use 4 spaces instead of tab it will be fixed.

I tried with spaces as well in task B and as here also but i don't know why that is happening.
even i tried to change in-front of proff. in class as well but that is not working in git somehow.

**Pull request merged**